erpnext/maintenance/doctype/maintenance_schedule/maintenance_schedule.py

```diff
@@ -58,9 +92,9 @@ def on_submit(self):
 
 			if no_email_sp:
 				frappe.msgprint(
-					frappe._("Setting Events to {0}, since the Employee attached to the below Sales Persons does not have a User ID{1}").format(
-						self.owner, "<br>" + "<br>".join(no_email_sp)
-				))
+						frappe._("Setting Events to {0}, since the Employee attached to the below Sales Persons does not have a User ID{1}").format(
+								self.owner, "<br>" + "<br>".join(no_email_sp)
+						))
```

Comment:
- Over-indentation
- Why `frappe._` when `_` is already imported in this file
```suggestion
						_("Setting Events to {0}, since the Employee attached to the below Sales Persons does not have a User ID{1}").format(
								self.owner, "<br>" + "<br>".join(no_email_sp)
						)
					)
```

Reply:
had auto-formatted previously, fixed